apsuite/commisslib/measure_bba.py

```diff
@@ -1562,7 +1562,8 @@ def make_figure_compare_bbas(
 
     # #### private methods ####
     def _do_bba(self):
-        self.data["bpms2dobba"] = self.bpms2dobba
+        if not self.data["bpms2dobba"]:
```

Comment:
I think this is prone to usage errors, isn't it. I think I understand what you wanted here, but maybe this is not the best way to do it. maybe you could read the measured_bpms list and append the bpms2dobba, what do you think?